app/views/companies/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1><%= @company.name %></h1>
 <p><%= @company.industry %></p>
-<p><%= @company.website %></p>
+<p><a href=<%= @company.website %> target=_blank><%= @company.website %></a></p>
```

Comment:
@jadijadi :+1: Thanks for your contribution. Can you wrap `<%= @company.website %>` in double quotes (as well as `_blank`)? That way a bad website value doesn't accidentally set other attributes on the tag. Another concern: it looks like [`target=_blank` is considered harmful](https://mathiasbynens.github.io/rel-noopener/) and avoiding it requires adding extra attributes to the anchor tag. As such I'm OK with using the current tab instead of opening a new one. Just like GitHub does. That said, thanks for the contribution.

Reply:
:+1: Thanks! LGTM.